src/Sylius/Component/Core/Uploader/ImageUploader.php

```diff
@@ -100,4 +100,16 @@ private function has(string $path): bool
     {
         return $this->filesystem->has($path);
     }
+
+    /**
+     * Will return true if the path is prone to be blocked by ad blockers
+     *
+     * @param string $path
+     *
+     * @return bool
+     */
+    private function isAdBlockingProne(string $path): bool
+    {
+        return stripos($path, 'ad') !== false;
```

Comment:
It filters out 11-12% of generated paths. Do adblockers block the images if `ad` is the directory name or also if `ad` is in the middle of a name?

Reply:
No need for case-insensitive check here though.

Reply:
As I mentioned in the intro I have not any ad blocker installed and I couldn't find any valid source  when trying to google it, so basically I just made the assumption that it would filter both paths and filenames